docs/ecosystem-user-event.mdx

```diff
@@ -7,24 +7,28 @@ title: user-event
 advanced simulation of browser interactions than the built-in
 [`fireEvent`](dom-testing-library/api-events.mdx#fireevent) method.
 
-[`user-event` on GitHub][gh]
+## Installation
 
-## Install
+With NPM:
```

Comment:
Maybe we can use a `Tab` for this one? :)

Reply:
What do you think about using #784 for that? It was closed, but personally I think it would be an easier solution.

Reply:
I agree that #784 has a nice and easy solution but I'm not sure we really need a third party package for something we usually do once in a long time.
Anyways, I don't have a strong opinion to either way so we can push that PR also if you think it's worth it.